src/sails_forth/http.clj

```diff
@@ -66,10 +66,13 @@
 (s/def ::access_token
   string?)
 
-(s/def ::authentication
+(s/def ::non-nil-authentication
```

Comment:
Eh, hate to bikeshed names, but specs maybe merit it... I guess I'd call this one ::authentication and I would use an anonymous spec `(s/nilable ::authentication) in the return value from `authentication`